client/resteasy-client/src/main/java/com/cerner/beadledom/client/resteasy/ExponentialBackoffHttpRequestRetryHandler.java

```diff
@@ -0,0 +1,49 @@
+package com.cerner.beadledom.client.resteasy;
+
+import java.io.IOException;
+import java.util.Random;
+import org.apache.http.impl.client.StandardHttpRequestRetryHandler;
+import org.apache.http.protocol.HttpContext;
+
+/**
+ * A {@link StandardHttpRequestRetryHandler} that adds an exponential back-off to retries.
+ *
+ * @author John Leacox
+ * @since 3.5
+ */
+class ExponentialBackoffHttpRequestRetryHandler extends StandardHttpRequestRetryHandler {
+  private static final int INITIAL_DELAY = 100;
+  private static final int BACKOFF_FACTOR = 2;
+  private static final Random random = new Random();
+
+  ExponentialBackoffHttpRequestRetryHandler(int retryCount, boolean requestSentRetryEnabled) {
+    super(retryCount, requestSentRetryEnabled);
+  }
+
+  @Override
+  public boolean retryRequest(IOException exception, int executionCount, HttpContext context) {
+    if (!super.retryRequest(exception, executionCount, context)) {
+      return false;
+    }
+
+    // Adding the delay here is not ideal, but there doesn't appear to be a better place in Apache
+    // httpclient similar to ServiceUnavailableRetryStrategy#getRetryInterval.
+    try {
+      Thread.sleep(calculateRetryDelay(executionCount));
+    } catch (InterruptedException e) {
+      // ignore
+    }
+
+    return true;
+  }
+
+  // VisibleForTesting
+  int calculateRetryDelay(int executionCount) {
```

Comment:
Should there be a maximum delay between requests?

Reply:
If we were implementing the full internal RFD on HTTP retries, then we would want a maximum, but my intent with this PR is to add a very simple retry backoff that follows the RFD but doesn't fully implement it. 

This class will only retry 3 times and is not configurable, so the maximum delay from the internal RFD can never be reached.